src/pilot/templates/network-environment.yaml

```diff
@@ -105,7 +107,34 @@ parameter_defaults:
   # CHANGEME: Change the following to the VLAN ID to use on the
   # External network
   ExternalNetworkVlanID: 190
-
```

Comment:
The MTU variables below (lines 110-133) should be in the nic_environment.yaml files, and should not appear here.